core/src/main/java/com/cloud/resource/ServerResource.java

```diff
@@ -31,6 +31,9 @@
  * ServerResource is a generic container to execute commands sent
  */
 public interface ServerResource extends Manager {
+
+    String ORIGINAL_FILE_EXTENSION = ".orig";
```

Comment:
@nvazquez where and how is this used?

Reply:
Thanks, was used on NfsSecondaryStorageResource only, moved it there